test/integration/createCoursesFile.test.js

```diff
@@ -1,5 +1,27 @@
-const test = require('tape')
+require('dotenv').config()
```

Comment:
Should these files be stored in the `integration` folder? Or should we create a `snapshot` folder for these kinds of tests instead? And run them with npm run test or npm run test:snapshot

Reply:
I don't have any opinions on this. I would consider those tests as _some kind of_ integration/regression test.

However the snapshots themselves are in a directory called `snapshots` under `integration` so be careful to not have files under a directory called `snapshot/snapshots` which lead to confusions